File: app/src/main/java/com/callmangement/model/fps_wise_complaints/ModelFPSComplaint.java
package com.callmangement.model.fps_wise_complaints;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class ModelFPSComplaint {
    @SerializedName("status") private String status;
    @SerializedName("message") private String message;
    @SerializedName("fPSComplainHistoryReqList") private List<ModelFPSComplaintList> fPSComplainHistoryReqList = new ArrayList<>();

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public List<ModelFPSComplaintList> getfPSComplainHistoryReqList() {
        return fPSComplainHistoryReqList;
    }

    public void setfPSComplainHistoryReqList(List<ModelFPSComplaintList> fPSComplainHistoryReqList) {
        this.fPSComplainHistoryReqList = fPSComplainHistoryReqList;
    }
}

File: app/src/main/java/com/callmangement/model/fps_wise_complaints/ModelFPSComplaint.kt
package com.callmangement.model.fps_wise_complaints

import com.google.gson.annotations.SerializedName

class ModelFPSComplaint {
    @SerializedName("status")
    var status: String? = null

    @SerializedName("message")
    var message: String? = null

    @SerializedName("fPSComplainHistoryReqList")
    private var fPSComplainHistoryReqList: List<ModelFPSComplaintList> = ArrayList()

    fun getfPSComplainHistoryReqList(): List<ModelFPSComplaintList> {
        return fPSComplainHistoryReqList
    }

    fun setfPSComplainHistoryReqList(fPSComplainHistoryReqList: List<ModelFPSComplaintList>) {
        this.fPSComplainHistoryReqList = fPSComplainHistoryReqList
    }
}
